packages/reactDom/src/ReactServerDOM.ml

```diff
@@ -1,11 +1,59 @@
 type json = Yojson.Basic.t
 
+module Env = struct
+  type t = DEV | PROD
+
+  let is_dev = function DEV -> true | PROD -> false
+  let to_string = function DEV -> "development" | PROD -> "production"
+end
+
+module Error = struct
+  type t = React.error
```

Comment:
I find this confusing. React.error is the client value, but what's the Error module in ReactServerDOM, a module that creates errors and turns them into json? Why t = React.error? Why Error.make_error?